src/main/scala/za/co/absa/spark/commons/test/SparkTestConfig.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spark.commons.test
+
+import java.util.TimeZone
+
+import org.apache.spark.sql.SparkSession
+
+trait SparkTestConfig {
+  def sparkSession: SparkSession
+  def timezone: TimeZone = TimeZone.getDefault
+  def appName: String = s"Commons unit testing"
```

Comment:
I believe it can be done much nicer and more modular:

```suggestion
  def sparkSession: SparkSession = {
    val result = builder.getOrCreate()
    timezone.foreach(tz => result.conf.set("spark.sql.session.timeZone", tz.getID))
    result
  }

  protected def builder: Builder = {
    SparkSession.builder()
      .master(master)
      .appName(appName)
  }
  protected def master: String
  protected def appName: String = s"Commons unit testing"
  protected def timezone: Option[TimeZone] = None
```